### DISCUSSION TITLE
CANT download instances

### DISCUSSION THREAD

**Discussion by wildepic2** [SKCraft/Launcher#510] [I need help!]
i am on 000 webhostapp and my client cant download instances from it

**Comment by hedgehog1029** [comment]
000webhost's free tier seems pretty limited from a cursorary glance - could I suggest [Netlify Drop](https://app.netlify.com/drop) instead?

We also have a [list of web hosts](https://github.com/SKCraft/Launcher/wiki/Website-Hosting) on the wiki.
